documentation/api/query/v4/pql.markdown

```diff
@@ -39,8 +39,8 @@ A PQL query has the following structure:
 
 Which is broken up into the following parts:
 
-* `entity`: Rquired. The entity context that this query executes on.
-* `projection`: Optional. Modifies the output of how the `entity` fields are returned.
+* `entity`: Required. The entity context that this query executes on.
+* `projection`: Optional. Restricts the output to a selection of fields.
```

Comment:
Not off the top of my head, I don't. 

So as I understand it, uh...

* A "filter" is a filter that controls which subset of entities the query will return. 
* A "projection" is one of the following:
    * A filter that modifies each entity in that subset.
    * Instructions on how to aggregate data in the subset of entities into a single value.

...? Right? This terminology is baffling to me as well. Maybe if you could explain the metaphor step by step? What's being projected, onto what surface?